File: problemas/iniciante/2543/2543.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada = input.trim().split("\n");

    let x = 0
    while (x < entrada.length) {
        let [N, L] = entrada[x++].trim().split(" ").map(Number);
        let soma = 0;

        for (let g = 0; g < N; ++g){
            let [i, j] = entrada[x++].trim().split(" ").map(Number);
    
            if (i === L && j === 0) {
                ++soma;
            }
        }
        console.log(soma);
    }

});

File: problemas/iniciante/2543/2543.py
def main():
    while True:
        try:
            N, L = map(int, input().split())
            soma = 0
            for _ in range(N):
                identificador, J = map(int, input().split())
                if identificador == L and J == 0:
                    soma += 1
            
            print(soma)

        except EOFError:
            break

if __name__ == "__main__":
    main()
